src/lightning_app/cli/lightning_cli.py

```diff
@@ -142,7 +155,7 @@ def run_app(
     app_args: List[str],
 ):
     """Run an app from a file."""
-    _run_app(file, cloud, without_server, no_cache, name, blocking, open_ui, env)
+    _run_app(file, cloud, cluster_id, without_server, no_cache, name, blocking, open_ui, env)
```

Comment:
can we raise if `cluster-id` is provided without `--cloud`

Reply:
done, and added unit tests for it.